ui/src/copainput.tsx

```diff
@@ -141,14 +171,33 @@ export function CopaInput(props: any) {
               <FormControl fullWidth>
                 <InputLabel id="trivy-verison-label">Trivy Version</InputLabel>
                 <Select
-                  label="Trivvyversion"
+                  label="Trivy Version"
+                  labelId='trivy-version-label'
                   onChange={(event: SelectChangeEvent) => {
                     setTrivyVerison(event.target.value as string);
                   }}
                 >
                   <MenuItem value={0}>Latest</MenuItem>
                 </Select>
               </FormControl>
+            </Box>
+            <Stack direction="row">
+              <FormControlLabel control={
+                <Switch
+                  checked={props.useContainerdChecked}
+                  onChange={handleLocalImageSwitchChecked}
+                />
+              } label="Containerd image store" />
+              <Tooltip title={"If you're scanning and patching an image that is local-only" +
+                " (i.e. built or tagged locally but not pushed to a registry), copa is limited" +
+                " to using docker's built-in buildkit service, and must use the containerd" +
+                " image store feature."}>
+                <IconButton onClick={() => {
```

Comment:
maybe we can make it more clear that the user would still have to enable it themselves and the switch doesn't enable it?